### DISCUSSION TITLE
License for commercial use

### DISCUSSION THREAD

**Discussion by vukasinpetrovic** [SixLabors/ImageSharp#1488] [Q&A] [answered]
Hi,
I stumbled upon ImageSharp library. I saw that it is currently under Apache-2 license. But I also saw that there is an option to buy a license on SixLabors website. I'm confused by that and I don't know if I can use this library for closed-source commercial projects or I need to buy a license for that purpose? Can someone explain that to me? Thanks in advance 😄

**Comment by JimBobSquarePants** [comment, marked as answer]
It’s an alternative license for commercial projects requiring support.

Apache 2.0 licenses the software AS-IS which means you are on your own should you require support in any form. This means that issues could likely be ignored, questions left unanswered etc. 

I hope that makes sense.

**Reply by vukasinpetrovic** [reply]
Got it. Thank you very much for quick answer.
